sql/core/src/main/scala/org/apache/spark/sql/execution/command/tables.scala

```diff
@@ -119,12 +119,9 @@ case class CreateTableLikeCommand(
 case class CreateTableCommand(table: CatalogTable, ifNotExists: Boolean) extends RunnableCommand {
 
   override def run(sparkSession: SparkSession): Seq[Row] = {
-    DDLUtils.verifyTableProperties(table.properties.keys.toSeq, "CREATE TABLE")
```

Comment:
I moved this check into `HiveExternalCatalog`. I think this is the limitation for hive metastore only. Our in memory metastore or our own metastore should not have this limitation.